### ISSUE TITLE
Facebook extremely slow to load

### ISSUE THREAD

**Issue by Drakhor** [dhowe/AdNauseam#1318]
I've been using AdNauseam on Chrome for quite some time now, and it worked fine up until a few days ago when Facebook, where it is mostly used, has become so slow that it is almost unresponsive. When I tried with another browser that didn't have AN installed, it loaded fine, so when I disabled it on Chrome, things went back to normal.

This has started about roughly 4-5 days ago, and at first I thought it was just FB having its usual issues, but it has only gotten worse.

- OS/version: Win 10 Pro
- Browser/version: Google Chrome Version 63.0.3239.132 (Official Build) (64-bit)
- AdNauseam version: 3.5.101
- Other extensions you have installed: A few, but the only ones that also alter my browsing experience are Abine TACO 1.50 and Unlimited Free VPN - Hola 1.80.99, which I don't use with FB, though.

**Comment by dhowe** [comment]
Would you mind creating a new profile in Chrome (under 'People' in Settings), installing only AdNauseam and trying again on the same page? Thanks

**Comment by Drakhor** [comment]
Done so, and the page loads a bit faster, but still slower than with the AN disabled. I will try again later during "prime time" to see how things are when more people are on at the same time.

**Comment by Drakhor** [comment]
Alright, so I've done some more testing and also asked around among other AN users and locals, and it seems AN is only partially responsible.
When there was more obvious slowdown, I've not only tested to see if solely AN is the cause, but I also ran some pings and tracert commands, and it turned out that there is a nearby French FB server (I'm located in Luxembourg) which tends to go down all the time, if only for a few minutes each time.
AN is still causing a slowdown, both on my normal Chrome profile and a clean one, but it's obviously not the main reason. It's also not that bad to a point where it would become an annoyance when everything is up and running fine.
I guess when I first tested with Chrome/AN and Edge/non-AN, I really must have hit time frames between that server going bonkers... -_-
